app/utils/database.py

```diff
@@ -410,6 +409,7 @@ class Individual(BaseModel):
     origin_herd = ForeignKeyField(Herd)
     name = CharField(50, null=True)
     certificate = CharField(20, null=True)
+    digital_certificate = IntegerField(sequence="certificates_seq", unique=True, null=True)
```

Comment:
I think this needs to be fixed to work with test as well, either by moving test to postgres or supporting sqlite, but I don't know exactly how yet.

I don't see that it needs to delay this specific PR, though (but should be fixed before merge to develop).